# Data Scientist

#### Technical Skills: Python, MATLAB, FORTRAN, MATHEMATICA, Julia, C#, SQL

## Work Experience

**Data Science Projects @ SUNY Binghamton (_August 2025 - Present)**


**Graduate Research Fellow @ Lawrence Berkeley National Lab (_June - August, 2024_)**
- Worked on a project of improving metagenome assembly scaffolding with machine learning.
- Comparison of different Large language models (LLMs) and their  applications to genomic datasets with different tokenization.

**Graduate Research Fellow @ Lawrence Berkeley National Lab (_June - August, 2023_)**
- Application of LLMs to perform downstream tasks (like classification, prediction) on genomic datasets.

## Education
- Ph.D., Mathematics | University of Alabama, Tuscaloosa (_2025_)								       		
- M.A., Mathematics	| University of Alabama, Tuscaloosa (_2022_)
- M.Tech., Industrial Mathematics | Federal Univ. Tech. Akure(_2019_)
- B.Tech., Industrial Mathematics | Federal Univ. Tech. Akure (_2014_)

## Selected Projects 

---
### [Machine Learning Application to Biochemical Kinetics](https://ieeexplore.ieee.org/abstract/document/10504344)

A machine learning framework called Nessie is applied to some notable biochemical systems to
generate the marginal probabilities that describe the evolution of those systems over time.

<img src="Images/Doc.png?raw=true"/>

[![](https://img.shields.io/badge/Julia-9558B2?logo=julia&logoColor=white)](https://julialang.org/)
[![](https://img.shields.io/badge/MATLAB-0076A8?logo=matlab&logoColor=white)](https://www.mathworks.com/products/matlab.html)
[![](https://img.shields.io/badge/LaTeX-008080?logo=latex&logoColor=white)](https://www.latex-project.org/)


---

### Improving Metagenome Assembly Scaffolding With Machine Learning


The project aims to determine machine learning approaches that can outperform the current heuristics in MetaHipMer (MHM). The learning approach can be integrated into the MHM pipeline in future work to offer improved assembly quality to all MHM users, with beneficial results for microbiome work, depending on the metagenome assembly. 

<img src="Images/CS_Poster.png?raw=true" src="Images/CS_Poster.png?raw=true" />

[![](https://img.shields.io/badge/Python-white?logo=Python)](#) [![](https://img.shields.io/badge/Jupyter-white?logo=Jupyter)](#) [![](https://img.shields.io/badge/PyTorch-white?logo=pytorch)](#) [![](https://img.shields.io/badge/HuggingFace_Transformers-white?logo=huggingface)](#)


---
### Method of Moments for Approximating Vascular Endothelial Growth Factor (VEGF) Receptor Models

We apply the method of moments to the VEGF receptor model, which describes the role of VEGF in the regulation of blood vessel formation
and growth through a series of molecular interactions between VEGF and its receptors on the surface of endothelial cells. The methodâ€™s accuracy
is evaluated by comparing the results with those obtained from the stochastic simulation algorithm and the chemical Langevin equation. The VEGF model is of interest because of the role of VEGF in the growth of cancer and other inflammatory diseases and the potential use of anti-VEGF therapies in the treatment of cancer.

<img src="Images/Vegf poster.png?raw=true"/>

[![](https://img.shields.io/badge/Python-white?logo=Python)](#) 
[![](https://img.shields.io/badge/MATLAB-0076A8?logo=matlab&logoColor=white)](https://www.mathworks.com/products/matlab.html)
[![](https://img.shields.io/badge/LaTeX-008080?logo=latex&logoColor=white)](https://www.latex-project.org/)
[![](https://img.shields.io/badge/Julia-9558B2?logo=julia&logoColor=white)](https://julialang.org/)

---
